chat-memories/spring-ai-jdbc-chat-memory/src/main/resources/org/springframework/ai/chat/memory/jdbc/schema-drop-mariadb.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS ai_chat_memory;
```

Comment:
Are these SQL scripts necessary for the JDBC chat memory? I see they are used in the integration test, but it's unclear how a user would use it. The docs below don't mention them either. Could you clarify? At the very least, we need to inform the end user of the existence of these scripts.

Reply:
I updated the docs and specified that the chat memory table will be automatically created using autoconfiguration and how to disable it. I also removed the sql files that are not used.